### ISSUE TITLE
Please add IFP, and TXD support!

### ISSUE THREAD

**Issue by fnavas00** [Parik27/DragonFF#83]
For the longest time I have been wanting to make scenes using the original animations from GTA San Andreas. Today I tried downloading Blender 2.57 along with the GTATools addon and some other addons that support TXD, IFP, etc. 

Long story short, it didnt work.

Anyhow..  ive been able to find these files. Which, somebody with some knowledge, could potentially use to 
add IFP and TXD support? maybe? no? yea I don't know a damn thing about coding. 

[import_2_pmd_bone_IK_skip_090713.zip](https://github.com/Parik27/DragonFF/files/6218696/import_2_pmd_bone_IK_skip_090713.zip)
[gtatools_for_blender_120812.zip](https://github.com/Parik27/DragonFF/files/6218697/gtatools_for_blender_120812.zip)
[GTA_IE_01c_export_menu_090713.zip](https://github.com/Parik27/DragonFF/files/6218698/GTA_IE_01c_export_menu_090713.zip)

**Comment by Parik27** [comment]
Hey,

It's not the lack of knowledge that's the reason for no TXD/IFP support. There are plans to add support for them in the future (they're mentioned as future features in the readme). However, at the moment (and it might continue for a few more months), I'd be unable to work on this project (apart from fixing major bugs or compatibility issues with future Blender versions). I had already started work on TXD support a while back and it is something that I'd definitely like to do.

Furthermore, you can use external tools for working with TXDs at the moment (like Magic.TXD). The intend for txd support in DragonFF is primarily to make it convenient to edit these files without having to use other tools (say automatic txd file creation based on the textures used in the Blender scene is something that I'd really like to implement and probably will in the future.), and also to facilitate map editing by removing the need for external tools like IMG editors and txd editors. However, the support that the tool will be able to provide will most likely be limited in abilities since it's not the primary focus of the tool (which is editing model files or map files).

About IFP support, you'd just have to wait for that or use other (older) Blender plugins at the moment. IFP isn't a priority at the moment, but I've had a couple of people ask for IFP support, so I plan to work on it after TXD and mapping support.

In conclusion, unless someone is willing to contribute these features in (data file reading is much more convenient in Python, and it doesn't have that much of a learning curve in my opinion), you'd have to wait at least a couple of months before you see any progress in TXD support. You can in the meantime, use external tools to help with TXD reading/writing. For IFP, you'll have to wait a little longer since I have plans to first finish TXD and mapping support.

Thanks.

**Comment by FAROK000** [comment]
Thank yu so much boss
